src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__day-1 calendar__day-1--start-day"></div>
```

Comment:
Modifiers should be on the main block
```suggestion
    <div class="calendar calendar--start-day--sun calendar--month--length-29">
      <div class="calendar__day"></div>
```